crates/sui-core/src/unit_tests/data/hero/sources/trusted_coin.move

```diff
@@ -16,7 +17,7 @@ module examples::trusted_coin {
     fun init(ctx: &mut TxContext) {
         // Get a treasury cap for the coin and give it to the transaction
         // sender
-        let treasury_cap = coin::create_currency<EXAMPLE>(EXAMPLE{}, 2, ctx);
+        let treasury_cap = coin::create_currency<EXAMPLE>(EXAMPLE{}, 2, b"EXAMPLE", b"", b"", option::none(), ctx);
```

Comment:
sam issue here

Reply:
This is unit test and it's up to the currency creator to decide whether to leave some fields empty(e.g, some coins might not need a description or logo)